homeassistant/components/bluetooth/manager.py

```diff
@@ -153,6 +153,12 @@ def supports_passive_scan(self) -> bool:
         """Return if passive scan is supported."""
         return any(adapter[ADAPTER_PASSIVE_SCAN] for adapter in self._adapters.values())
 
+    def async_scanner_count(self, connectable: bool = True) -> int:
+        """Return the number of scanners."""
+        if connectable:
+            return len(self._connectable_scanners)
+        return len(self._connectable_scanners) + len(self._scanners)
```

Comment:
Maybe not related to this PR, but the the attribute naming seems confusing if it's correct that `self._connectable_scanners` are connectable scanners and `self._scanners` are non-connectable scanners. Based on the naming, I would expect `self._scanners` to be all scanners, not only the non-connectable ones.

Reply:
Renamed in 96534fa086